### PR TITLE
new server user reference docs

### PR DESCRIPTION
**Pull request by JacKaL37**
hey my guy, more deets for peeps

### CODE REVIEW

**Comment by JacKaL37** [comment]
@jonmatthis check it outtttttt

File: Writerside/s.tree [hunk 1]
```diff
@@ -109,4 +109,12 @@
         <toc-element topic="FileSystem.md"/>
         <toc-element topic="TimedEvents.md"/>
     </toc-element>
+    <toc-element topic="User-Guides.md">
+        <toc-element topic="Discord-Server-User-Reference.md">
+            <toc-element topic="Server-Structure.md"/>
+            <toc-element topic="Discord-Actions.md"/>
+            <toc-element topic="AI-Bot-Interactions.md"/>
```
Comment (jonmatthis) [inline Writerside/s.tree:116]: tf is this?!

File: Writerside/s.tree [hunk 2]
```diff
@@ -109,4 +109,12 @@
...
+            <toc-element topic="AI-s-Context-Window.md"/>
+        </toc-element>
```
Comment (jonmatthis) [inline Writerside/s.tree:118]: sd

File: Writerside/topics/AI-Bot-Interactions.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## 🔮🤖 - AI Bot Interactions
+Ways you can interact with the bot within discord.
+
+- 🔮 -  `/chat` spawns a new chat `thread`, with a `text:` parameter you can put something in to get started.
+  - 🗨️ - `sending` a `message` will trigger the bot to process its `context window` and produce a `response` which it will gradually `stream` by updating its response message. 
+  - 🗒️ - a note for long AI messages: if it runs over the message length, it will chop it off and start a fresh message, but also attach the complete message response as a `text file`, which you can either `download` or just click to expand it and read the raw text. 
+  - 🚫 - a `~` at the start of your message in an AI chat thread means the bot won't notice a message, so you can make a quick note, or @ tag me to come have a look at something specific, whatever you like. 
+
+- 📎 - attachment types supported:
```
Comment (jonmatthis) [inline Writerside/topics/AI-Bot-Interactions.md:12]: we actually call these buts

File: Writerside/topics/AI-Bot-Interactions.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  - 🎙️ - `voice messages` will first be transcribed into `text`, and then sent off to the LLM, enabling a much more naturalistic way of speaking your thoughts. 
+    - On `mobile` you can do this natively, on `desktop` you need to attach a file you recorded separately (most OSs have an app for this natively)
+  - 📃 - `text files` will be added to the `context window` along with your message.
+
+- ⚙️  - adjust the `prompt settings` for your AI conversations:
```
Comment (jonmatthis) [inline Writerside/topics/AI-Bot-Interactions.md:17]:
```suggestion
- ⚙️  - Adjust the `prompt settings` for your AI conversations:
```

File: Writerside/s.tree (hunk 1 above)
Comment (JacKaL37) [reply Writerside/s.tree:116]: is good. u no worry

**Review by jonmatthis — APPROVED** [review]

File: Writerside/topics/AI-Bot-Interactions.md (hunk 3 above)
Comment (JacKaL37) [reply Writerside/topics/AI-Bot-Interactions.md:12]: u do not

**Pull request merged**